PYME/LMVis/Extras/pointSetGeneration.py

```diff
@@ -105,6 +105,37 @@ class ImageSource(PointSource):
     #name = Str('Density Image')
     #foo = Enum([1,2,3,4])
 
+    helpInfo = {
+        'points_per_pixel' : '''
+Select average number of points (dye molecules or docking sites) per pixel in the image mask.
```

Comment:
mask is technically inaccurate here as the image is a density image, not necessarily a binary one. Acceptance probability = image*points_per_pixel.